calendario/forms.py

```diff
@@ -36,7 +39,7 @@ class TurnoForm(forms.ModelForm):
 
     servicio = forms.ModelChoiceField(
         label='Servicios',
-        queryset=Servicio.objects.all(),
+        queryset=Servicio.objects.filter(),
```

Comment:
@ramosfm Acá no habría que pasarle el estado?
```suggestion
        queryset=Servicio.objects.filter(estado=EST_ACTIVO),
```

Reply:
si esta bien, igual lo hace mas adelante en el iniciar formulario, ahi puede qeudar el filter q pusiste vos o el all. Gracias!

Reply:
Servicio [no tiene estado](https://github.com/cluster311/ggg/blob/develop/centros_de_salud/models.py#L83-L88). Creo que hubo una confusión.
O quizás yo no estoy viendo algo

Reply:
Estoy tratando de entender por qué propuse ese cambio. Creo me confundí con Centros de Salud.